docs/topics/proc_show-debug-logging-openshift-spring-boot.adoc

```diff
@@ -0,0 +1,91 @@
+
+= Show Debug Logging on OpenShift
+
+Use the the `JAVA_DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+.Prerequisites
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+.Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ mvn clean fabric8:deploy -Popenshift
+----
++
+This command uses the Fabric8 Maven Plugin to launch the link:{link-s2i-process}[S2I process] on OpenShift and to start the pod.
+
+. Find the name of your pod and follow the logs to watch it start.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+....
+$ oc logs -f pod/POD_NAME
+----
++
+IMPORTANT: After your pod has started, keep watching the logs.
+
+. Test your application.
++
+For example, if you had debug logging in the xref:mission-http-api-spring-boot[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. Return to your pod logs and notice there are no debug logging messages in the logs.
+
+. Set the `JAVA_DEBUG` environment variable to enable debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get dc
+...
+$ oc set env dc DC_NAME JAVA_DEBUG=mybooster
+----
+
+. Return to your pod logs to watch the update roll out.
++
+IMPORTANT: After the update has rolled out, keep watching the logs. This is where the debug logging will appear.
+
+. Test the application to invoke debug logging.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get routes
+...
+$ curl $APPLICATION_ROUTE/api/greeting?name=Sarah
+----
+
+. View the application logs to see the debug messages.
++
+[source,options="nowrap",subs="attributes+"]
+----
+...
+mybooster name: Sarah +3m
```

Comment:
Again, this message is copied&pasted from Node.js guide, with Spring Boot, it will be very different.

Reply:
@Ladicek thanks for the comments will have a SME review and update expected output.